File: python/bc1149.py
line = list(map(int, input().split(" ")))
cVal = []
so = 0

if line[1] > 0:
    cVal.append(line[0])
    cVal.append(line[1])
else:
    for c in range(1, len(line)):
        if line[c] > 0:
            cVal.append(line[0])
            cVal.append(line[c])
            break

for c in range(0, 2):
    if c == 0:
        for d in range(0, cVal[0] + 1): so += d
    else:
        for d in range(0, cVal[c]): so += d

print(f"{so}")

File: cpp/bc1149.cpp
#include <iostream>
#include <sstream>

using namespace std;

int main()
{
    string line, token;
    int i = 0, fVal, sVal, res = 0;
    
    getline(cin, line);
    
    int tam = line.size() / 2 + 1;
    int cVal[tam];
    
    stringstream ss(line);

    while(ss >> token)
    {
        cVal[i] = stoi(token); 
        i++;
    }
    
    fVal = cVal[0];
    
    if(cVal[1] > 0) sVal = cVal[1];
    else
    {
        for(int j = 2; j < tam; j++)
        {
            if(cVal[j] > 0)
            {
                sVal = cVal[j];
                break;
            }
        }
    }
    for(int j = 1; j < fVal+1; j++) res += j;
    for(int j = 0; j < sVal; j++) res += j;
    
    cout << res << endl;
    return 0;
}

